controllers/configmap_controller.go

```diff
@@ -340,3 +340,51 @@ func (r *ConfigMapReconciler) createServicesConfigMap(ctx context.Context, names
 	}
 	return windowsServices, nil
 }
+
+// EnsureServicesConfigMapExists ensures that the ServicesConfigMap is present and valid on the cluster during operator
+// bootup. ConfigMapReconciler.createServicesConfigMap() cannot be called in its stead as the cache has not been
+// populated yet, which is why the typed client is used here as it calls the API server directly.
+func (r *ConfigMapReconciler) EnsureServicesConfigMapExists() error {
+	var windowsServices *core.ConfigMap
+	var err error
```

Comment:
You can include the return variable (`err`) explicitly in the signature of the function.